### PR TITLE
[WOR-521] Deprecate v1 billing endpoints, clean up v2 endpoints

### PR DESCRIPTION
**Pull request by cahrens**
Based on changes in https://github.com/broadinstitute/rawls/pull/2046/files#diff-6e80a735b01ebcbdc253d755a6078cc1af5a585a6bd04b92b6455a9469988dff

Tested out with locally running firecloud-orchestration.

### CODE REVIEW

File: src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -377,7 +405,7 @@ paths:
           content:
             'application/json':
               schema:
-                $ref: '#/components/schemas/BillingProjectResponse'
+                $ref: '#/components/schemas/RawlsBillingProjectResponse'
```
Comment (cahrens) [inline src/main/resources/swagger/api-docs.yaml:408]: The return value will conform to this once https://github.com/broadinstitute/rawls/pull/2047 is merged.

File: src/main/resources/swagger/api-docs.yaml [hunk 2]
```diff
@@ -2734,7 +2763,8 @@ paths:
     get:
       tags:
         - Profile
-      summary: List billing projects for a user
+      summary: List billing projects for a user. Deprecated, use v2 billing API.
+      deprecated: true
```
Comment (cahrens) [inline src/main/resources/swagger/api-docs.yaml:2767]: These "profile" methods actually dispatch to the `user/billing` (v1) endpoints.

image

File: src/main/resources/swagger/api-docs.yaml [hunk 3]
```diff
@@ -8064,6 +8114,50 @@ components:
...
+          deprecated: true
+          type: string
+          description: informational message about the project
+        azureManagedAppCoordinates:
+          $ref: '#/components/schemas/AzureManagedAppCoordinates'
+        cloudPlatform:
+          type: string
+          enum:
+            - GCP
+            - AZURE
```
Comment (cahrens) [inline src/main/resources/swagger/api-docs.yaml:8160]: Probably want to add "UNKNOWN" here too. Will see what @blakery adds in the Rawls PR.

**Review by blakery — APPROVED** [review]

File: src/main/resources/swagger/api-docs.yaml (hunk 3 above)
Comment (cahrens) [reply src/main/resources/swagger/api-docs.yaml:8160]: Added.

**Review by marctalbott — APPROVED** [review]

**Pull request merged**